### ISSUE TITLE
Horos 4.0.0 Plugins Issue

### ISSUE THREAD

**Issue by DrAMokhtari** [horosproject/horos#720]
hi guys I have Horos 4.0.0 for MacBook M1 and I installed plugins for Horos but I can't use any as you see here
<img width="596" alt="image" src="https://user-images.githubusercontent.com/119264156/204332889-799d6c07-7f57-484b-a794-7dae06a8151a.png">
<img width="523" alt="image" src="https://user-images.githubusercontent.com/119264156/204332996-1c2da3ea-91b9-47d9-bd68-2553ba7579c1.png">

**Comment by jensen0914** [comment]
It may be a macos permissions/security block. Have you tried this workaround?

https://groups.google.com/g/horos-project/c/5mMpNg2lNn4/m/XwmuYGjfAQAJ

If you launch Horos from a Terminal window and monitor the output you will see errors for unsigned plugins.

Todd Jensen, PhD
Jensen Informatics LLC

**Comment by DrAMokhtari** [comment]
yes I tried it but nothing changed, I says it's installed but I can't use it, is the issue with the 4.0.0RC5_Apple version ?

**Comment by DrAMokhtari** [comment]
it worked when I downgraded to the 3.3.6 version

**Comment by jensen0914** [comment]
Ah, if you use the Apple 4.0.0RC5_Apple version, which is only compiled for Apple Silicon (aka M1, M2 chips), the plugins must also be compiled for Apple Silicon. Most plugins were only compiled for Intel chips, not Apple Silicon or both (aka Universal).

If you download the 4.0.0RC for Intel version the plugins should work.

**Comment by DrAMokhtari** [comment]
Can I find please the list of plugins which are compatible with the M1 ship please?
